web_external/IndexView/style.styl

```diff
@@ -39,7 +39,7 @@ fieldset
   flex-flow row
 
   .left-sidebar
-    width 280px
+    width 290px
```

Comment:
I would still very much prefer using font units here...

Reply:
Again, with the latest browser, I don't see benefit using em over px.